tests/ui/higher-rank-trait-bounds/issue-59311.stderr

```diff
@@ -4,15 +4,15 @@ error: higher-ranked lifetime error
 LL |     v.t(|| {});
    |     ^^^^^^^^^^
    |
-   = note: could not prove `[closure@$DIR/issue-59311.rs:17:9: 17:11] well-formed`
+   = note: could not prove `where [closure@$DIR/issue-59311.rs:17:9: 17:11] well-formed`
```

Comment:
this is strange, though more of a side-effect of us not having surface syntax for WF goals